shared/setupTests.ts

```diff
@@ -0,0 +1,9 @@
+import { restoreConsoleMethods, throwErrorOnConsoleMethods } from "../consoleMethodsSetupAndBreakdown";
+
```

Comment:
Question, who do we have this file in shared and web?

Reply:
It's because we have different setups for web and shared